frame/lending/src/tests.rs

```diff
@@ -269,22 +270,15 @@ fn accrue_interest_plotter() {
 #[test]
 fn test_borrow_repay_in_same_block() {
 	new_test_ext().execute_with(|| {
-		let collateral_amount = 900000;
+		let collateral_amount = 900000000;
 		let (market, vault) = create_simple_market();
-		// Balance for ALICE
-		assert_eq!(Tokens::balance(MockCurrencyId::USDT, &ALICE), 0);
 		assert_ok!(Tokens::mint_into(MockCurrencyId::USDT, &ALICE, collateral_amount));
-		assert_eq!(Tokens::balance(MockCurrencyId::USDT, &ALICE), collateral_amount);
 
 		assert_ok!(Lending::deposit_collateral_internal(&market, &ALICE, collateral_amount));
 		assert_eq!(Tokens::balance(MockCurrencyId::USDT, &ALICE), 0);
 
-		// Balance of BTC for CHARLIE
-		// CHARLIE is only lender of BTC
```

Comment:
why remove these checks? it has meaning there

Reply:
we must not test other pallet here. mint works. also limit is stream not from balance but from deposits/transfers, so we do not care initial amount -> mint works here too. we should mix what tests should and should not test

Reply:
for me it feels natural to see if user has received tokens or not. I think better to consult @KaiserKarel @hussein-aitlahcen

Reply:
mint_if_zero_before_and_ensure_minted_what_was_requested